phabfive/maniphest.py

```diff
@@ -81,10 +96,149 @@ def create_from_config(self, config_file, dry_run=False):
                     has_errors = True
 
             if has_errors:
-                log.error(f"Could not find one or more specified projects in phabricator instance. All available project names")
+                log.error(f"Could not find one or more specified users, projects in phabricator instance. All available project names")
                 log.error(list(project_name_to_id_mapping.keys()))
                 return (False, None)
 
+        ### Validate owner and all subscribers (=users) is correct data types and exists in Phabricator instance
+        #create a new construct to consolidate all users for owner and subscriber
+        users_query = {}
+        users_query["data"] = []
+
+        r = self.phab.user.search()
+        length = len(r.data)
+        afterCursor = r['cursor']['after']
+        users_query["data"].extend(r.data)
+
+        while (length >= 100 and afterCursor is not None):
```

Comment:
So this might not need to be done in this PR yet but one idea for this kind of hack that seems to be needed all over in general, we could either do a simple helper function that wraps the cursor logic with a yield and a few more dynamic stuff for what endpoint to search, but then that code could be reused more efficiently.

In the long run i think we might want to go to the more Django Model based solution where things are wrapped within the logic of the models where we would sort this kind of problem in there and the end-user up here would never know. This however is for a future version and not within this PR